File: c++/menu.cpp
#include "menu.h"
#include <string>
#include <iostream>

using namespace std;

Menu::Menu() {
}

Menu::~Menu() {
    cout << "Instance removed." << endl;
}

int Menu::get_food_cost(string food) {
    int cost = 0;
    if (food == "meal")
        cost = 5000;
    else if (food == "special_meal")
        cost = 7000;
    else if (food == "coke")
        cost = 2000;
    else if (food == "cidar")
        cost = 2000;
    else
        cost = 0;
        
    return cost;
}

File: c#/Menu.cs
using System;

public class Menu {
    public Menu() {
    }

    ~Menu() {
        System.Console.WriteLine("Instance removed.");
    }

    public int get_food_cost(string food) {
        switch (food) {
            case "meal":
                return 5000;
            case "special_meal":
                return 7000;
            case "coke":
                return 2000;
            case "cidar":
                return 2000;
            default:
                return 0;
        }
    }
}
